ospfd/ospf_vty.c

```diff
@@ -8309,6 +8309,8 @@ DEFUN (ip_ospf_hello_interval,
 	struct ospf_if_params *params;
 	params = IF_DEF_PARAMS(ifp);
 	uint32_t seconds = 0;
+	bool is_addr = false;
+	uint32_t old_interval = 0;
```

Comment:
nit: unnecessary initialization

Reply:
Yes initialisation might not be required here but it helps to fix the code Coverity issues . I think here it is no harm with initialisation.